### ISSUE TITLE
irregular P-P interval

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9604]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
"Multifocal atrial tachycardia (MAT) is defined as irregular P-P, R-R, and P-R intervals,
isoelectric baseline between P waves, and ventricular rate over 100 beats/min"
PMID 35813391

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA 3282 Multifocal atrial tachycardia (MAT)

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
irregular P-P (etc.) does not seem to be an independent phenotype, e.g., 

Sinus Arrhythmia, or "regularly irregular" sinus rhythm, is a variation on sinus rhythm where the P-P interval ( the distance between consecutive P waves) varies by more than 10%. 

Would it be sufficient to add a term for Multifocal atrial tachycardia
def:  A supraventricular tachycardia with a rapid, irregular atrial rhythm arising from multiple ectopic foci within the atria. This arrhythmia is characterized by a heart rate of greater than 100 beats per minute with organized atrial activity yielding three or more different non-sinus P-wave morphologies in the same lead. There are irregular PP intervals and an isoelectric baseline between P waves

PMID: 29083603

**Comment by carolina-f** [comment]
I am not a specialist in cardiology, but looking for clinical signs of MAT, I have found the same definition in many scientific articles:

PMID: 29441751  Multifocal atrial tachycardia (MAT), also known as chaotic atrial arrhythmia, is rare and 
arises from multiple foci of increased automaticity located within the atria. MAT is defined 
by the presence of 3 or more P wave morphologies, irregular P-P, R-R, and P-R intervals, an 
isoelectric baseline between the P waves, and ventricular rate over 100 beats/min.

PMID: 29441747  Multifocal atrial tachycardia (MAT) is defined as a tachycardia with at least 3 morphologically 
distinct P-waves. It is also characterized by irregular P-P, R-R, and P-R intervals, and an 
isoelectric baseline between the P-waves.

PMID: 35813391 Multifocal atrial tachycardia (MAT) is defined as the presence of 3 or more P wave morphologies,
which is characterized by irregular P-P, R-R, and P-R intervals, isoelectric baseline between the P
waves, and ventricular rate over 100 beats/min on the electrocardiogram

PMID: 17051756 MAT is characterized by electrocardiographic findings including: (1) multiple (at least 3) distinct P-wave morphologies; (2) irregular P–P intervals; (3) isoelectric baseline between P-waves; and (4) occasionally rapid ventricular rate

PMID: 11499730 The condition is characterized by the electrocardiographic (ECG) finding of: 1) multiple (at least three)
distinct P-wave morphologies; 2) irregular P-P intervals; 3) isoelectric baseline between P-waves; and 4) ventricular rate100 beats/min

**Comment by pnrobinson** [comment]
So you mean you want to enter the individual features of MAT, rather than MAT as a kind of independent finding?

**Comment by carolina-f** [comment]
To correctly characterize the phenotype of the disease, I would need each of the signs of the Electrocardiogram

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001087
- label: Irregular P-P interval
- parents: [Abnormal EKG (HP:0003115)](https://hpo.jax.org/app/browse/term/HP:0003115)
- pmid: PMID:35813391
- def: Abnormally increased variability in the timing between consecutive P-waves on an electrocardiogram.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9951
- synonyms:
